README.md

```diff
@@ -23,9 +23,13 @@ skip that if you want to run it from the repl.
 
 Kaocha runner exposes four commands:
 
+- `kaocha-runner-run-test`
+
+   Runs the current test in the current namespace.
```

Comment:
I guess it's the test at point? It could be made slightly clearer. :)

Reply:
you are abs. right. will rename in both files.